### ISSUE TITLE
how to build

### ISSUE THREAD

**Issue by InvisibleRasta** [zzossig/hugo-theme-zzo#246]
hello i have been trying to build my website using the zzo theme . I followed the guide on the zzo page: https://zzodocs.netlify.com/docs/gettingstarted/quickstart/
when launching hugo server, everything works as expected but when i just run hugo and build the website and click public/index.html there is no theme displayed.just a page saying Example domain.
wha ti would like to do is use the default example site and modify on top of it

**Comment by fandean** [comment]
I think it may be related to your directory structure and where you run the command

**Comment by kfrajer** [comment]
Wow I was having the exact same issue. I don't think the **index.html** in the root folder is setup or there is some tweaking that still needs to be done (and it is not in the _quickstart_)

When I deploy the site using `hugo -d public`, it runs fine locally @ `https://localhost/1313`

When I deployed in gihub pages, I had to access the site via `https://kfrajer.github.io/en/index.html`.

Then reading your post made me realized that there is a setting for the url set as default to be "example.com". This is the url in the **config.toml** file inside the `/config/_default` folder.
